site/src/pages/TaskPage/TaskPage.tsx

```diff
@@ -221,7 +221,29 @@ const WorkspaceNotRunning: FC<WorkspaceNotRunningProps> = ({ workspace }) => {
 		? mutateStartWorkspace.error
 		: undefined;
 
-	return (
+	const deleted = workspace.latest_build?.transition === "delete" as const;
+
+	return deleted ? (
+		<Margins>
+			<div className="w-full min-h-80 flex items-center justify-center">
+				<div className="flex flex-col items-center">
+					<h3 className="m-0 font-medium text-content-primary text-base">
+						Task workspace was deleted.
+					</h3>
+					<span className="text-content-secondary text-sm">
+						This task can't continue. Delete this task and create a new one.
```

Comment:
```suggestion
						This task cannot be resumed. Delete this task and create a new one.
```
Terminology -> status match.